src/justfile.rs

```diff
@@ -472,6 +472,7 @@ mod tests {
       recipe,
       line_number,
       code,
+      ..
```

Comment:
It's more verbose, but I think I'd extract and check these with an assert below:

```suggestion
      suppress_error_message,
```